packages/story-editor/src/components/canvas/karma/quickActions.karma.js

```diff
@@ -502,7 +502,7 @@ describe('Quick Actions integration', () => {
       // The "Replace background media" quick action also triggers a help center tip which steals focus.
       // Open the tip now to avoid the focus stealing after the quick menu button is clicked.
       const { actions } = await fixture.renderHook(() => useHelpCenter());
-      actions.openToUnreadTip.text(KEYS.ADD_BACKGROUND_MEDIA);
+      actions.openToUnreadTip(KEYS.ADD_BACKGROUND_MEDIA);
```

Comment:
Looks like this was failing. Looking at the help center provider it seems that `openToUnreadTip` is a function without a `.text` property.